### ISSUE TITLE
my files or example files since this week not working

### ISSUE THREAD

**Issue by markus1971** [BerlinPix/filo#6]
hi,
last week, i made a download of all files, tested = working.
now, when i open my galleries or your example files it`s not working.
i see only the loading bar.

thanks
markus

**Comment by markus1971** [comment]
i think, it was a temporary facebook bug :-)
now all working!

**Comment by markus1971** [comment]
very unstable!
today the script not working, online and your examples.
why?
markus

**Comment by BartFra** [comment]
I have the same issue right now. Have you found a solution yet?

**Comment by markus1971** [comment]
Sorry, no.
One day ok, next fail.
Tested it in different pages.

Von meinem iPhone gesendet

**Comment by BerlinPix** [comment]
There is a problem with the Facebook Graph API and so the tool is as you said unstable. I'm working on that. please be have some patience until I fix the bug.

**Comment by markus1971** [comment]
Danke für die Info.
VG
Markus



Zitat von BerlinPix <notifications@github.com>:

**Comment by markus1971** [comment]
Gibt es eigentlich einen Grund, wieso die "alte" Version von früher  
nicht mehr funktioniert?
Die lief ja stabil.

VG
Markus




Zitat von BerlinPix <notifications@github.com>:

**Comment by BerlinPix** [comment]
I answering in english so everyone can follow...

Facebook had deactivated FQL which was used in older versions of filo.js. FQL was replaced by the Graph API but this API has more restrictions than FQL. 

https://developers.facebook.com/docs/reference/fql/ 

Almost every request to the Graph API expects a valid access token. I've generated one for filo.js but obviously Facebook limited the amount of requests per token per day/ or hour. After a lot of users updated to the new version there are lot of more request using that token.

So one quick fix would be to use your own token as written in the readme.

https://github.com/BerlinPix/filo/#access-token

I'm working on that but I don't have a better solution yet. I will let you know if I've found another way.

**Comment by markus1971** [comment]
with the own access token it works without error :-)

another problem:
i have 2 albums ("Lichtblicke Gala" and "Photos Lichtblicke Gala").
with this source code both albums are shown.
if i change the name of the album to Photos LB Gala, only the one choosen album will shown.

<div id="js-filojs"></div><div class="filo" data-facebook-id="myid" data-max-count="100"><div class="filo__album" data-album-name="Lichtblicke Gala"></div>

**Comment by erik-wendt** [comment]
Do you have a code example? (without the token ;-))

**Comment by markus1971** [comment]
`<div` id="js-filojs"></div><div class="filo" data-facebook-id="myid" data-max-count="100"><div class="filo__album" data-album-name=Lichtblicke Gala"></div>`

**Comment by erik-wendt** [comment]
Take look at [this example](https://github.com/BerlinPix/filo/blob/master/examples/12-filter.html).

```
<div class="filo" data-facebook-id="filojs" data-max-count="2">
    <div class="filo__album" data-album-name="new.*" ></div>
    <div class="filo__album" data-album-name="lost.*"></div>
</div>
```

When filo is loading your albums from Facebook, it looks for any album name wich matches to your album name. This was a requested feature in the past. 

So in this example filo will load every album which starts with "new" or "lost". But I see the point. Maybe I should add the possibility to define if the matching should be used or not.

**Comment by markus1971** [comment]
it`s a cool feature but in some cases not beneficial.





Zitat von Erik Wendt <notifications@github.com>:

**Comment by erik-wendt** [comment]
will be fixed with #7
